apps/pipelines/migrations/0006_pipelinechathistory_pipelinechatmessages.py

```
@@ -0,0 +1,44 @@
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PipelineChatHistory',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('type', models.CharField(choices=[('node', 'Node History'), ('named', 'Named History'), ('global', 'Global History'), ('none', 'No History')], max_length=10)),
+                ('name', models.CharField(db_index=True, max_length=128)),
+                ('session', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='pipeline_chat_history', to='experiments.experimentsession')),
```

Ruff

Line too long (166 > 120)

(E501)